recipe/meta.yaml

```diff
@@ -1,30 +1,79 @@
-{% set version = "1.0.3" %}
+{% set version = "1.0.4" %}
 
 package:
   name: c99-to-c89
   version: {{ version }}
 
 source:
-  fn: c99-to-c89-{{ version }}.zip  # [win]
-  url: https://github.com/libav/c99-to-c89/releases/download/release-1.0.3/c99-to-c89-{{ version }}.zip  # [win]
-  md5: d58bc9741f258841aa3196a1b4155e51
+  - url: https://github.com/AnacondaRecipes/c99-to-c89/archive/release-1.0.4.tar.gz
+    sha256: 1df2ec430d34e9034d162969fe1a12cee50fa67cacdcc60d7b57b6c45b576467
+  # We need a libclang. No one builds them statically for Windows unfortunately (though the
+  # libav binaries have been built this way). I rename the libclang.dll in these to avoid
+  # conflicts with conda-forge's clangdev (which we cannot use at present due to compat).
```

Comment:
@mingwandroid, what were the issues with conda-forge's clangdev?

Reply:
Conda-forge's clangdev is much too modern. I spent a while figuring out the latest official release that still worked OK. The semantics have changed considerably in the cursor / code walking stuff (and I spent *waaaay* too long on this package to try to modernise that part of it, please feel free if you are interested!)

While we could rummage around in old clangdev packages trying to find one that will work, I'd rather not force old clangdevs into the build environment incase of conflicts so I elected to just bundle an official DLL in a non-conflicting location instead.

Reply:
But .. why do you ask this? I wrote what I thought was a comprehensive explanation in the comment:

https://github.com/conda-forge/c99-to-c89-feedstock/pull/6/files#diff-e178b687b10a71a3348107ae3154e44cR13

.. did you miss that or did it not make sense or something?